lib/service/service.go

```diff
@@ -840,8 +840,11 @@ func initExternalLog(auditConfig services.AuditConfig) (events.IAuditLog, error)
 			loggers = append(loggers, logger)
 		case dynamo.GetName():
 			hasNonFileLog = true
+			endpoint := uri.Query().Get(teleport.Endpoint)
```

Comment:
Can you add a `SetFromURL` method to `dynamoevents.Config` similar to `firestoreevents.EventsConfig`? Then `SetFromURL` can be used to override the endpoint value?

Reply:
Done :)